package.json

```diff
@@ -14,7 +14,6 @@
     "@theforeman/builder": "^8.4.1",
     "@theforeman/eslint-plugin-foreman": "^8.4.1",
     "@theforeman/find-foreman": "^8.4.1",
-    "@theforeman/stories": "^8.4.1",
     "@theforeman/test": "^8.4.1",
     "@theforeman/vendor-dev": "^8.4.1",
```

Comment:
@evgeni can you try to change the packages accepted version to ">= 8.4.1" ?

Reply:
done, and now?

Reply:
thanks, now `npm test -- -u` to fix the tests snapshots

Reply:
I don't have a dev setup locally where I could run that. Sorry.
(and my change should not have changed any snapshots?)

Reply:
I'm guessing it happens through upgrade of an package, so it's probably broken in main (that's why I hate snapshoting btw :stuck_out_tongue: )

Reply:
I'd love to fix the snapshots, but no dev setup here either :(